filebeat/input/file_other.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 type FileStateOS struct {
-	Inode  uint64 `json:"inode,omitempty"`
-	Device uint64 `json:"device,omitempty"`
+	Inode  uint64 `json:"inode,"`
```

Comment:
how about adding an 'os' field for sanity check? So when reading config from another OS (e.g. windows) Inode will be 0, but 'os' field will be populated with 'windows'.